.github/workflows/build-main.yml

```
@@ -38,6 +38,11 @@ jobs:
       - name: Test
         run: yarn test
 
+      - name: Deploy Storybook
+        run: yarn run deploy-storybook --ci --source-branch=main
```

```suggestion
        run: yarn run deploy-storybook -- --ci --source-branch=main
```
no? Not sure for Yarn but with NPM we have to add `--` before the list of parameters.

---

It was, but I saw a deprecated message when I put it and yarn is warning that it's not useful

---

Okay thx!